imagick.c

```diff
@@ -610,7 +610,7 @@ static zval *php_imagick_read_property(zend_object *object, zend_string *member,
 					if (format) {
 						retval = rv;
 						ZVAL_STRING(retval, format);
-						php_strtolower(Z_STRVAL_P(retval), Z_STRLEN_P(retval));
+						zend_str_tolower(Z_STRVAL_P(retval), Z_STRLEN_P(retval));
```

Comment:
I bet it should be used conditionally depending on PHP version
```suggestion
#if PHP_VERSION_ID < 80400
						php_strtolower(Z_STRVAL_P(retval), Z_STRLEN_P(retval));
#else
						zend_str_tolower(Z_STRVAL_P(retval), Z_STRLEN_P(retval));
#endif
```

Reply:
Other 2 places also needs this

Reply:
Why - it is present at least since PHP 5.4 - https://github.com/php/php-src/blob/PHP-5.4.45/Zend/zend_operators.h#L332

Reply:
Sorry, I didn't check php-src, looks like the function exists, so probably it's not required